janus_server/Cargo.toml

```diff
@@ -28,6 +28,7 @@ prio = { git = "https://github.com/divviup/libprio-rs", rev = "229cd9c45924c7dae
 rand = "0.8"
 reqwest = { version = "0.11.4", default-features = false, features = ["rustls-tls", "json"] }
 ring = "0.16.20"
+runtime-hpke = { git = "https://github.com/jbr/runtime-hpke/" }
```

Comment:
Could we cut a release of runtime-hpke? libprio-rs is currently at a specific commit rather than a released version, but that's because we're not the only team using libprio-rs -- I don't suppose that applies to runtime-hpke.

If we can't cut a release version, let's pin to a specific commit revision as libprio-rs does.

Reply:
Released as runtime-hpke@0.1.0 under my own account